solr/core/src/test/org/apache/solr/cloud/ReplicationFactorTest.java

```diff
@@ -461,38 +447,18 @@ protected void doDelete(UpdateRequest req, String msg, int expectedRf, int retri
 
   protected int sendDoc(int docId, int minRf) throws Exception {
     UpdateRequest up = new UpdateRequest();
-    boolean minRfExplicit = maybeAddMinRfExplicitly(minRf, up);
     SolrInputDocument doc = new SolrInputDocument();
     doc.addField(id, String.valueOf(docId));
     doc.addField("a_t", "hello" + docId);
     up.add(doc);
-    return runAndGetAchievedRf(up, minRfExplicit, minRf);
+    return runAndGetAchievedRf(up);
```

Comment:
Alright, so with `maybeAddMinRfExplicitly` gone and the `minRf` argument in `runAndGetAchievedRf` gone the `minRf` argument of `sendDoc` becomes unused, right? I wonder how the code would work out if `sendDoc` had its `minRf` argument taken away too? Bit like peeling an onion this here i.e. multiple layers (but it hopefully won't make anyone cry).

Reply:
The tests for `ReplicationFactorTest` and `HttpPartitionTest` passed on my end after removing the `minRf` argument from `sendDoc`. I committed the `ReplicationFactorTest` changes separately as it meant removing the `expectedRf` argument from the `sendDoc` call in `addDocs` and I wasn't sure if this was OK, even though the test still passed.

Reply:
I appreciate you committing the changes separately, thanks. The `ReplicationFactorTest` change looks good to me.